ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -70,6 +71,10 @@ export default function OnboardingFlow() {
   const isFromReminder = new URLSearchParams(search).get('isFromReminder');
   const trackEvent = useContext(MetaMetricsContext);
 
+  useEffect(() => {
```

Comment:
can we add a comment on the top of this use effect to explain why we have it here ?